lib/nice_partials/monkey_patch.rb

```diff
@@ -24,12 +24,19 @@ def with_nice_partials_t_prefix(lookup_context, block)
   end
 end
 
+require "active_support/deprecation"
+NicePartials::DEPRECATOR = ActiveSupport::Deprecation.new("1.0", "nice_partials")
```

Comment:
@andrewculver not sure what version number we want in here, could be something in the `0.x` range too. Your pick 😄

Reply:
1.0 is good. That’s where we’re heading for sure.